libpod/container_inspect.go

```diff
@@ -630,16 +631,7 @@ type InspectNetworkSettings struct {
 }
 
 // Inspect a container for low-level information
```

Comment:
Maybe extend the comment that this method expects callers to have locked the container (if needed)